Cargo.toml

```diff
@@ -26,6 +26,7 @@ madara_cli_types = { path = "crates/types" }
 
 # External dependencies
 anyhow = "1.0.82"
+ci_info = "0.14.14"
```

Comment:
In a next PR this can be removed by replacing this logic with envs as config source.
In docker compose you can specify an .env file which compose will source before running the containers